internal/device/cpu_power_meter.go

```diff
@@ -24,6 +24,10 @@ type EnergyZone interface {
 	// When energy usage reaches this value, the energy value returned by Energy()
 	// will wrap around and start again from zero.
 	MaxEnergy() Energy
+
+	// Power() returns the current power consumption by the zone.
+	// This method is used for zones that provide instantaneous power readings.
+	Power() (float64, error)
```

Comment:
would be beneficial to add a method to return the type/kind? i.e. a method which returns the units or type of unit (Energy/ Power) . because the name of the go type still remains `EnergyZone`